package.json

```diff
@@ -26,21 +26,20 @@
   ],
   "license": "OGL-UK-3.0",
   "dependencies": {
-    "@hapi/boom": "^7.4.2",
```

Comment:
I take it boom is an explicit dependency of @hapi/hapi so does not need to be specified?

Reply:
We just don't seem to reference it anywhere.